R/antigen.garnish_run.R

```diff
@@ -171,6 +171,135 @@ ag_data_err <- function(){
 
 }
 
+#' Internal function to configure netMHC suite of tools
+#'
+#' @param dir Character vector. Path to `antigen.garnish` data directory.
+#'
+#' @export configure_netMHC_tools
+#' @md
+
+configure_netMHC_tools <- function(dir){
+
+  owd <- getwd()
+
+  on.exit({setwd(owd)})
+
+  # netMHC parent dir in ag data dir
+  npd <- file.path(dir, "netMHC")
+
+  if (!dir.exists(npd)) dir.create(npd)
```

Comment:
The script should exit if the directory doesn't exist (no subdirectories can exist)